internal-services/catalog/file-updates-process-file-updates-task.yaml

```diff
@@ -77,9 +80,12 @@ spec:
 
           echo -e "=== UPDATING ${REPO} ON BRANCH ${REVISION} ===\n"
 
-          cd /tmp/
+          cd "${TEMP}"
           git_clone_and_checkout --repository "${REPO}" --revision "${REVISION}"
 
+          # updating local branch with the upstream
+          git_rebase -n "glab-base" -r "${UPSTREAM_REPO}" -v "${REVISION}"
```

Comment:
Sorry for being late, but what if there is a conflict in the rebase? Why is rebase even needed? Shouldn't we always just clone the upstream repo and start from there?